cmd/skaffold/app/cmd/apply.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2021 The Skaffold Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/spf13/cobra"
+)
+
+// NewCmdApply describes the CLI command to apply manifests to a cluster.
+func NewCmdApply() *cobra.Command {
+	return NewCmd("apply").
+		WithDescription("Apply hydrated manifests to a cluster").
+		WithExample("Hydrate Kubernetes pod manifest first", "render --output rendered-pod.yaml").
+		WithExample("Then create resources on your cluster from that hydrated manifest", "apply rendered-pod.yaml").
+		WithCommonFlags().
+		WithHouseKeepingMessages().
+		WithArgs(func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return errors.New("`apply` requires at least one manifest argument")
+			}
+			return nil
+		}, doApply)
+}
+
+func doApply(ctx context.Context, out io.Writer, args []string) error {
```

Comment:
should you allow `glob` arguments like `kubectl` deployer currently does?

Reply:
should we support urls?

Reply:
should  we support reading from `STDIN` so I could do this 😅:
In `examples/multi-config-microservices`
```
skaffold diagnose --yaml-only | skaffold render -f - | skaffold apply -
```

Reply:
haha yes these are all great suggestions 😄  i think i'm fine with supporting all 3 of these, but how about i add them in a follow up PR? they seem like QOL enhancements to this feature